apps/console/src/features/applications/pages/application-edit.tsx

```diff
@@ -197,6 +200,25 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
         getApplication(id);
     }, []);
 
+    /**
+    * Fetch the identity provider id & name when calling the app edit through connected apps
+    */
+    useEffect(() => {
+        if(typeof history.location.state !== "object"){
+            return;
+        }
+
+        setisConnectedAppsRedirect(true);
+        type idpInfoType = {
+            "id": string, 
+            "name": string
+        };      
+        const idpInfo: { "id": string, "name": string } = history.location.state as idpInfoType;
```

Comment:
Use "idpInfoType" interface here instead of refining it.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3494/commits/3478ccc81887593d88f43d6c9659d6c0d0c10ed6